nifi-api/src/test/resources/TestVariableRegistry/foobar.properties

```diff
@@ -0,0 +1 @@
+fake.property.3=test me out 3, test me out 4
```

Comment:
I already added the rat check but I can add the ASF license, that's not a problem.